### ISSUE TITLE
MathJax not readable to Screen Readers

### ISSUE THREAD

**Issue by sam-lee-ku** [mathjax/MathJax#3381]
Hello, I've been trying to get MathJax to be read by screen readers, specifically NVDA. The math renders just find, but as soon as I tab into the math itself - it reads as "clickable math" which when I space/enter into it, reads as Math interaction not supported.

I proceeded to switch our version of tex-svg to tex-svg-full, but still have not been able to resolve the issue. Can I get some help on how to resolve this?

**Comment by dpvc** [comment]
The speech support has been updated in v4.0, and I think it will address your problem, so Im closing this for now.
